awx/main/conf.py

```diff
@@ -913,26 +915,56 @@
 
 
 def logging_validate(serializer, attrs):
-    if not serializer.instance or not hasattr(serializer.instance, 'LOG_AGGREGATOR_HOST') or not hasattr(serializer.instance, 'LOG_AGGREGATOR_TYPE'):
+    needed_attrs = (
+        'LOG_AGGREGATOR_HOST',
+        'LOG_AGGREGATOR_TYPE',
+        'LOG_AGGREGATOR_PROTOCOL',
+    )
+    if not serializer.instance or not all(hasattr(serializer.instance, attr) for attr in needed_attrs) or not attrs.get('LOG_AGGREGATOR_ENABLED', False):
```

Comment:
This is messing me up. I'm trying to manually run your test cases, but essentially all cases are bailing early on this line and not throwing the 400 errors because of that.

Still not sure to do with this information, but that's the big thing for verifying the functionality of this PR.